src/frontend/components/design-system/molecules/drop-zone/drop-zone.tsx

```diff
@@ -24,11 +24,21 @@ const validateSize = (
   return +maxSize >= +size
 }
 
-const inKb = (size: string | number): string => {
+const inUnit = (size: string | number, unit: FileSizeUnit = 'kb'): string => {
```

Comment:
It would be awesome if FieSizeUnit can be automatically guessed by the file size: if its above 1000kb - print 'mb'